src/components/elements/default-tab.js

```diff
@@ -121,12 +121,13 @@ const Tabs = ({
     mobile_font_size,
     mobile_tab_button_underline_length,
     has_no_query,
+    starting_index = 0,
```

Comment:
just a suggestion:
instead of passing the index, you can use the value of the tab from tab_list. because this way, if they decide to change the order of tab items, you don't need to change the logic here again.